pkg/util/log/log.go

```diff
@@ -286,10 +299,12 @@ func (sw *DatadogLogger) errorStackDepth(s string, depth int) error {
 }
 
 // critical logs at the critical level, called with sw.l held
-func (sw *DatadogLogger) critical(s string) error {
+func (sw *DatadogLogger) critical(s string, logger seelog.LoggerInterface) error {
 	scrubbed := sw.scrub(s)
 	err := sw.inner.Critical(scrubbed)
```

Comment:
Incomplete migration to the "logger" version, you're correctly calling `logger.Critical` below, but you're also still calling `sw.inner.Critical`, so this looks like it will log twice in the standard case.